src/videodownlinkbandwidthpolicy/VideoPriorityBasedPolicyConfig.ts

```diff
@@ -34,4 +44,80 @@ export default class VideoPriorityBasedPolicyConfig {
     }
     this.networkIssueRecoveryDelayFactor = networkIssueRecoveryDelayFactor;
   }
+
+  private static readonly MINIMUM_DELAY = 2000; 
```

Comment:
Nit: I understand why this is put down here but lets just put all these constants and members above the constructor for consistency with code base.